### DISCUSSION TITLE
youtube age restiction

### DISCUSSION THREAD

**Discussion by kozunoki** [jagrosh/MusicBot#795] [Questions]
is there a way to login or bypass the youtube age restriction on the bot? I can't play certain songs due to the restriction ex: youtube login to continue

**Comment by YJSoft** [comment]
It should be fixed / implemented from lavaplayer.(JMusicBot does nothing about youtube)

**Comment by MichailiK** [comment]
As I mentioned in <https://github.com/jagrosh/MusicBot/issues/753#issuecomment-918088113>, IPs/users within the European Economic Area (meaning all of Europe, Switzerland etc.) have a stricter age gate. You **may** be able to get around it by hosting your bot in the US or another continent.

Logging into YouTube [won't be added](https://github.com/jagrosh/MusicBot/wiki/Things-That-Won%27t-Be-Added#features-requiring-additional-api-keys). The bot is kept to be simple, without any credentials (apart from a Discord bot token of course).

**Comment by CyberFlameGO** [comment]
Hey there, this is actually a pull-requested feature upstream over on lavaplayer. See https://github.com/sedmelluq/lavaplayer/pull/649
